app/models/imports/episode_rss_import.rb

```diff
@@ -110,7 +107,7 @@ def entry_description(entry)
   end
 
   def entry_description_attribute(entry)
-    [:content, :itunes_summary, :description, :title].find { |d| !entry[d].blank? }
+    [:content, :description, :itunes_summary, :itunes_subtitle, :title].find { |d| !entry[d].blank? }
```

Comment:
content and description consistently have the full html version much more often than summary